nyxx/lib/src/core/guild/Guild.dart

```diff
@@ -287,6 +287,11 @@ class Guild extends SnowflakeEntity {
   Future<GuildSticker> fetchSticker(Snowflake id) =>
       client.httpEndpoints.fetchGuildSticker(this.id, id);
 
+  /// Fetches all roles that are in the server.
+  /// To get a single file use .firstWhere()
```

Comment:
I dont think that information is needed here, fist line would be sufficient

Reply:
All sorted in latest push